src/modules/expense/expenseFormSlice.tsx

```diff
@@ -0,0 +1,86 @@
+// src/features/expenseForm/expenseFormSlice.js
+import { createSlice } from '@reduxjs/toolkit';
+import dayjs, { Dayjs } from 'dayjs';
+
+export const expenseFormSlice = createSlice({
+  name: 'expenseForm',
+  initialState: {
+    category: [],
+    textValue: '',
+    selectedWallet: '',
+    showAttachmentComponent: false,
+    showContinueButton: false,
+    switchChecked: false,
+    toggleContinueButton: false,
+    selectedFrequency: '',
+    selectedEndAfter: '',
+    showPopup: false,
+    showRepeatDetails: false,
+    date: dayjs('2022-04-17'),
+    showSummarizeSection: false,
+    repeatSectionContinueButton: false,
+  },
+  reducers: {
+    setCategory: (state, action) => {
```

Comment:
Remove unnecessary reducers after updating data in the store

Reply:
Bu kısmı anlayamadım Anna, tüm reducerlar' ım gerekli değil mi? Bir iki tane örnek reduces söyleyebilir misin kaldırılması gereken?

Reply:
Yukarıda tekrar Redux'ta tutulması gerekmeyen veri işaretledim. Onları sildikten sonra bu kısmı da düzeltmek gerekir. Mesela `setShowAttachmentComponent` silmen gerekecek.